src/run/run.c

```diff
@@ -637,23 +647,58 @@ static int transient_unit_set_properties(sd_bus_message *m, UnitType t, char **p
 }
 
 static int transient_cgroup_set_properties(sd_bus_message *m) {
+        _cleanup_free_ char *slice = NULL;
+        _cleanup_free_ char *name = NULL;
         int r;
         assert(m);
 
-        if (!isempty(arg_slice)) {
-                _cleanup_free_ char *slice = NULL;
+        if ((int) !!arg_slice + (int) !!arg_slice_inherit_set > 1)
```

Comment:
why not `if (arg-slice && arg_slice_inherit_set)`? Would be more readable to me...

Reply:
copied from other argument handling style, but sure.